tests/config_edge_test.py

```
@@ -0,0 +1,77 @@
+def test_load_config_invalid_toml():
+    """Test handling of invalid TOML syntax."""
+    invalid_toml = b"""
+[incomplete
+missing closing bracket
+"""
+    with tempfile.NamedTemporaryFile(mode="wb", suffix=".toml", delete=False) as f:
+        f.write(invalid_toml)
+        f.flush()
+
+        try:
+            with pytest.raises(Exception):  # Should raise a TOML parsing error
```

Ruff (0.13.2)

Do not assert blind exception: `Exception`

(B017)